app/main.py

```diff
@@ -1,2 +1,56 @@
+from collections.abc import Hashable
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
```

Comment:
These attrs shouldnt be parametrized - it's part of dict internal structure. Also use Fraction for load factor and use correct value - 2/3